.github/sync-repo-settings.yaml

```diff
@@ -5,24 +5,6 @@ branchProtectionRules:
 - pattern: master
   isAdminEnforced: true
   requiredStatusCheckContexts:
-    - 'ci/circleci: docs'
```

Comment:
should we add the actions CI here?

Reply:
I was under the impression that sync-repo-settings.yaml was generated based on settings saved in the UI and was not intended to be hand edited. @busunkim96 can you confirm or correct?

Reply:
@alexander-fenster is correct, the names of the new actions CI should be added here so sync-repo-settings bot preserves it as a required checks. 

The file is added to repos that have non-default CI configurations and overrides the default settings.

Reply:
Done